src/components/SelectAvatar/index.js

```diff
@@ -0,0 +1,50 @@
+import React, { useState } from 'react';
+import { Image } from 'react-native';
+import { func } from 'prop-types';
+import ImagePicker from 'react-native-image-picker';
+
+import Button from 'components/common/Button';
+import defaultProfileImage from 'assets/images/defaultProfileImage.png';
+import strings from 'locale';
+import { BLACK } from 'constants/colors';
+import styles from './styles';
+
+const {
+  PROFILE: { selectAvatar },
+} = strings;
+
+const options = {
+  title: selectAvatar,
+  storageOptions: {
+    skipBackup: true,
+    path: 'images',
+  },
+};
+
+const SelectAvatar = ({ setAvatarData }) => {
+  const [avatarSource, setAvatarSource] = useState('');
+
+  const onSelect = () =>
+    ImagePicker.showImagePicker(options, response => {
+      if (response.data) {
```

Comment:
I think you need to check  `if (!response.didCancel && !response.error)` here and then you can destructure `data` and `uri`